boards/arc/axs/support/openocd.cfg

```diff
@@ -0,0 +1,73 @@
+#  Copyright (C) 2019 Synopsys, Inc.
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+# Configure JTAG cable
+# SDP has built-in FT2232 chip, which is similar to Digilent HS-1, except that
+# it uses channgel B for JTAG, instead of channel A.
+interface ftdi
+ftdi_vid_pid 0x0403 0x6010
+ftdi_layout_init 0x0088 0x008b
+ftdi_channel 1
+
+
+adapter_khz 10000
+
+# ARCs supports only JTAG.
+transport select jtag
+
+
+#
+# HS Development Kit SoC.
+#
+# Contains quad-core ARC HS38.
```

Comment:
AXS 103 is not really a quad core. As far as I know, it can be configured as either a HS38 dual core, or as a single HS36/34.
Current initial support is for HS38 dual core mode, right?

Reply:
currently, SMP on axs103 not supported. I will clarify it.

Reply:
It is still a bit confusing and inconsistent. This openocd config contains multiple CPU references/names: EM, HS38, L2 cache, etc. while the dts file has a single hs48.
Please make consistent. If for the openocd file the specific processor is a don't care, then use some generic name, like HS3x/4x, or simply ARC HS or so.